src/redux/data.js

```diff
@@ -85,6 +86,16 @@ import {
 } from '@rainbow-me/utils';
 import logger from 'logger';
 
+const storage = new MMKV();
+
+function addHiddenCoins(coins, dispatch, address) {
+  const storageEntity = storage.getString('hidden-coins-' + address);
```

Comment:
Is it worth extracting `'hidden-coins-' + address` into a variable called `storageKey` since it's also used to set the value?

Reply:
w/e , I added `storageKey` :)